linen_examples/imagenet/imagenet_fake_data_benchmark.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Benchmark for the ImageNet example using fake data for quick perf results."""
+import pathlib
+import time
+
+from absl import flags
+from absl.testing import absltest
+from absl.testing.flagsaver import flagsaver
+import imagenet_main
+from configs import default as default_lib
```

Comment:
did you consider adding a testing config?

Reply:
Good idea, thanks!